suite-native/app/android/gradle.properties

```diff
@@ -25,7 +25,7 @@ android.useAndroidX=true
 android.enableJetifier=true
 
 # Version of flipper SDK to use with React Native
-FLIPPER_VERSION=0.162.0
+FLIPPER_VERSION=0.164.0
```

Comment:
Can you please revert this Flipper related changes before merge?

Reply:
Yeah, I will